### PR TITLE
Support multiple formats for test reporting

### PR DESCRIPTION
**Pull request by nigurr**
This address the issue where the reporting will be visible both in Console as well as Junit reporting xml

### CODE REVIEW

**Comment by nigurr** [comment]
@stephenmichaelf please review this.
This address the issue where the reporting will be visible both in Console as well as Junit reporting xml

File: package-lock.json [hunk 1]
```diff
@@ -142,13 +142,13 @@
...
       "dev": true,
       "requires": {
-        "cache-base": "1.0.1",
-        "class-utils": "0.3.6",
-        "component-emitter": "1.2.1",
-        "define-property": "1.0.0",
-        "isobject": "3.0.1",
-        "mixin-deep": "1.3.1",
-        "pascalcase": "0.1.1"
+        "cache-base": "^1.0.1",
```
[the remark this answers is not in the document]
Comment (nigurr) [reply package-lock.json]: no idea. all the packages which are added static versions.

File: make.js [hunk 2]
```diff
@@ -343,7 +343,8 @@ target.test = function() {
     // setup the version of node to run the tests
     util.installNode(options.node);
 
-    run('mocha ' + testsSpec.join(' ') /*+ ' --reporter mocha-junit-reporter --reporter-options mochaFile=../testresults/test-results.xml'*/, /*inheritStreams:*/true);
+    var configFile = path.join(__dirname, 'testconfig.json');
+    run('mocha ' + testsSpec.join(' ') + ' --reporter mocha-multi-reporters --reporter-options configFile=' + configFile + '', /*inheritStreams:*/true);
```
[the remark this answers is not in the document]
Comment (nigurr) [reply make.js]: file will always exist as we are shipping this config in task code.

**Comment by stephenmichaelf** [comment]
@nigurr What is the state of this?

**Pull request closed without merging**